data/ai/BA/maphandler.lua

```diff
@@ -17,6 +17,9 @@ local function EchoDebug(inStr)
 	end
 end
 
+local map
+local game
```

Comment:
map and game might give different results for different teams, so remove these 2 bits and switch the others like the `self.ai` part